pywr/parameters/parameters.py

```diff
@@ -130,6 +130,27 @@ def load(cls, model, data):
 InterpolatedVolumeParameter.register()
 
 
+class InterpolatedLevelParameter(AbstractInterpolatedParameter):
+    """
+    Generic interpolation parameter for a non-storage node. It interpolates a value based on node flow at previous timestep
```

Comment:
Docstring should contain one sentence first, and then a paragraph with a more detailed explanation. We try to follow the numpy docstring guidance: https://numpydoc.readthedocs.io/en/latest/format.html